src/tilemap/TileLayer.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Threading.Tasks.Dataflow;
+
+namespace Tile_Mapper.tilemap
+{
+    //Class to store and manipulate information related to tile layers
+    public class TileLayer
+    {
+        //Variables
+
+        private string tileSet;
```

Comment:
Please follow the C# guidelines, private variables start with an underscore. Here it would be `_tileSet`.